src/app.js

```diff
@@ -34,15 +34,15 @@ app.get("/", (req, res) => {
 });
 
 app.get("/service-description", (req, res) => {
-  res.render("serviceDescription.njk"); // free text
+  res.render("serviceDescription.njk", { ga4ContainerId: "GTM-KD86CMZ" }); // free text
```

Comment:
Do you think because we're using the GA4 tag we could create a config file and import this in maybe? Just thinking if we need to change the variable, would be handy to change it in one place :D Let me know what you think

Reply:
@Joanneyoung01 True. It's maybe the good occasion to create a constant and store it in a constants.js file

Reply:
@Joanneyoung01 @di-fabs ok , I have  now created a constants file and imported the variable into the render methods